src/Sylius/Behat/Page/Shop/Product/ShowPage.php

```diff
@@ -61,10 +61,10 @@ public function addToCartWithVariant(string $variant): void
 
     public function addToCartWithOption(ProductOptionInterface $option, string $optionValue): void
     {
-        $select = $this->getDocument()->find('css', sprintf('select#sylius_add_to_cart_cartItem_variant_%s', $option->getCode()));
+        $select = $this->getDocument()->find('css', sprintf('[data-test-item-variant-%s]', $option->getCode()));
```

Comment:
All data like code, name, etc we should put in attribute value for example: ``[data-test-item-variant="%s"]``
here and in other places.